### ISSUE TITLE
[PROD] Early Warning Field Report not displaying correct fields/data

### ISSUE THREAD

**Issue by nanometrenat** [IFRCGo/go-frontend#985]
## Issue
This is separate to #984 which relates only to "actions taken".  This ticket tracks some other issues with *Early Warning* field report type.

**Viewing submitted Field Report** (https://dsgoproxyapp.azurewebsites.net/reports/9454)
The following fields don't pull through to the View mode at all:

- Forecasted Date of Impact
- Information Bulletin

Also NB I foolishly didn't input a Potentially Affected Province, not sure how that comes through.

The following fields didn't pull through data correctly:

- Planned International Response - there are 5 items in form, but only 4 pull through (Forecast Based Action is missing), and labels don't match input.  e.g. for Emergency Appeal I had selected "Launched" but view mode shows as "Active".  For Emergency Response Unit I had selected "Planned", but there is no section for ERU in the view mode, however there's a section called IFRC Staff which is showing as "Planned".

![image](https://user-images.githubusercontent.com/33625192/75781144-fc549f00-5d5c-11ea-909a-44e49b21ddbd.png)
![image](https://user-images.githubusercontent.com/33625192/75781188-0c6c7e80-5d5d-11ea-9ff4-2f78938f2ba5.png)

- Contact section is missing phone numbers (though other contacts data is showing correctly)

The following fields pull through data correctly but have different labels in View vs what was originally input - @mmusori can you please confirm if these are OK as-is or need to be aligned?

- "Affected Pop Centres" vs "Largest Population Centres Likely to be Affected"
- "Sources for data marked as Other" vs "Source Details"
- "Description" vs "Risk Analysis" (strongly suggest this one should be aligned and say Risk Analysis in both places, for Early Warning type)
- "Assisted by Government" and "Assisted by RCRC Movement" vs "Assisted by Government - Early Action" and "Assisted by RCRC Movement - Early Action"
- in the actions section, the Input view shows "Early Actions Taken by NS", "Early Actions taken by IFRC", "Early Action taken by other RCRC Movement", "Early Actions Taken by Others (Governments, UN)" whereas the view mode doesn't mention "Early" in the label.
- in actions section NB there's also slight difference in wording for the rest of the Movement label and Government label (view mode shows "Actions taken by any other RCRC Movement actors" and "Actions taken by others") but still same meaning so may not be a big issue the captions are different.

**Edit mode for Field Report** (https://dsgoproxyapp.azurewebsites.net/reports/9454/edit)

- contacts - Title and Phone don't display at all (although Name and Email pull through correctly)
- actions don't pull through at all (this bit is tracked via 984)
![image](https://user-images.githubusercontent.com/33625192/75779354-bfd37400-5d59-11ea-9101-71b575f1a400.png)
![image](https://user-images.githubusercontent.com/33625192/75779370-c661eb80-5d59-11ea-8e66-3534dde0f730.png)

**NOTE** Some fields from the Field Report should also pull through to the Emergency - I haven't checked the Emergency at all.

## Steps to reproduce
Create a new field report (event type or early warning type), click through all screens, submit. Then view onscreen (in both view and edit mode on the front end) and check that the various sections are pulling through and are labelled correctly.

## Expected behaviour
Whatever data I submitted in the process of creating the Field Report should display (and display with the same selections) when I go back and view or edit the information.

## Related feature
Early Warning Field Report type was introduced in #833
Other tickets re fields appearing etc. for Field Reports include #895, #896 and #984 

## Impact
Those submitting field reports

## Severity
Not all submitted information is displayed.

## Is there a workaround?
Workaround for some fields (look in a different screen) but not for other fields.

**Comment by batpad** [comment]
@nanometrenat I have addressed the critical issues in this ticket in this PR: https://github.com/IFRCGo/go-frontend/pull/990 - the pull request contains more details of exactly what has been fixed.

Out of the list of items where you have tagged @mmusori I have changed the `Description` label for the Early Warning Field Report, but not done anything on the other items. All other items in the ticket should be addressed.
